keras_cv/bounding_box/converters.py

```diff
@@ -22,6 +25,69 @@ class RequiresImagesException(Exception):
     pass
 
 
+def _encode(
```

Comment:
Even though this is an internal API lets give this a more descriptive name.

Something like: "encodes_boxes_to_deltas"

Reply:
Thanks for the suggestion. Done